### DISCUSSION TITLE
GLIBC 2.34 error

### DISCUSSION THREAD

**Discussion by ashokkumar2012** [mulesoft-labs/data-weave-native#78] [General]
Getting this error on the latest Debian. Is dw binary only supported on ubuntu?

 ./dw --help
./dw: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by ./dw)

 more /etc/issue
Debian GNU/Linux 11 \n \l

**Comment by machaval** [comment]
Hi @ashokkumar2012, if you have a debian instance you can try to build it from sources. That may be the easiest way to validate. Currently we are building and testing the CLI on Ubuntu instances.
